habitat-lab/habitat/isaacsim/remote_content/converter.py

```diff
@@ -0,0 +1,54 @@
+''' 
+This module takes converts glb files to usd. 
+
+The system that runs this folder needs have the Issac Lab repo installed.
+The converter function from isaac lab can import only after the Isaac Lab stage
+is launched. asyncio library is used because the source code for 
+MeshConverter._convert_mesh_to_usd is also an async function. 
+
+'''
+import asyncio
+import argparse
+from omni.isaac.lab.app import AppLauncher
+import os
+import timer
+
+#Run laucher through Isaac Lab
+
+
+async def convert_glb_to_usd(glb_folder_path: str, usd_folder_path: str) -> None:
```

Comment:
Let's not create AppLauncher inside the function, because we'll likely call this function several times inside a larger application in the future. You can create the AppLauncher in your `if __name__ == "__main__":` block so that the script still works standalone.

Reply:
Applauncher class moved.